server/dashboards/preconfigured/bot-framework-datepicker.ts

```diff
@@ -0,0 +1,1146 @@
+/// <reference path='../../../client/@types/types.d.ts'/>
+import * as _ from 'lodash';
+
+// The following line is important to keep in that format so it can be rendered into the page
+export const config: IDashboardConfig = /*return*/ {
+  id: 'bot_analytics_dashboard',
```

Comment:
on new templates, id/url properties should be changed, this will be irrelevant if you updated one of the bot framework templates.